src/index.js

```diff
@@ -2,7 +2,6 @@
 import './style.css';
 import arrayPokemonLinks from './apiFunctions.js';
 import { fillPopUp, submitComment, getComments } from './popup.js';
-import { pokemon } from './fake_pokemon.js';
```

Comment:
We can safely delete the fake_pokemon.js file since we're no longer using it 😄

Reply:
That line is already deleted.